lib/tasks/migrations/plans/load_dc_benefit_market_catalog.rake

```diff
@@ -55,9 +55,34 @@ namespace :load do
 
         if result.success?
           p "Success: created eligibility for #{effective_date.year} benefit market catalog"
+
+          puts "::: Creating SHOP OSSE eligibilities"
+
+          count = 0
+          ::BenefitSponsors::BenefitSponsorships::BenefitSponsorship.where(:"eligibilities.key" => "aca_shop_osse_eligibility_#{calender_year}".to_sym).each do |benefit_sponsorship|
+            count += 1
+            start_on = effective_date - 1.day
+            eligibility = benefit_sponsorship.eligibility_for("aca_shop_osse_eligibility_#{start_on.year}".to_sym, start_on)
+            osse_eligibility = eligibility.blank? ? false : eligibility.is_eligible_on?(start_on)
+
```

Comment:
do we need to still CreateShopOsseEligibility when  osse_eligibility is false?

Reply:
yes